sdks/java/core/src/main/java/org/apache/beam/sdk/coders/StructuredCoder.java

```diff
@@ -33,18 +32,23 @@
  * <p>To extend {@link StructuredCoder}, override the following methods as appropriate:
  *
  * <ul>
- *   <li>{@link #getComponents}: the default implementation returns {@link #getCoderArguments}.</li>
- *   <li>{@link #getEncodedElementByteSize} and
- *       {@link #isRegisterByteSizeObserverCheap}: the
+ *   <li>{@link #getComponents}: the default implementation returns {@link #getCoderArguments}.
+ *   <li>{@link #getEncodedElementByteSize} and {@link #isRegisterByteSizeObserverCheap}: the
```

Comment:
Some flavor on the relationship of these with registering an element byte size observer would be welcome here.